syntax_body/java/syntax/pattern.vim

```diff
@@ -31,3 +31,23 @@ sy match  javaInclude       '\v<import%(\_s+static)=>'
 sy match  javaPackagePath   '\v<%(%(\w|\$)+\_s*\.\_s*)*%(\w|\$)+>'
 \   contained contains=javaIdentifier,javaOperator
 "---------------------------------------------------------------------------------------------------
+if !exists("java_ignore_javadoc") && main_syntax != 'jsp'
+  " syntax coloring for javadoc comments (HTML)
+  syntax include @javaHtml <sfile>:p:h/html.vim
```

Comment:
This command can be replaced by the following command without the default `html.vim`.
```suggestion
  syntax include @javaHtml syntax/html.vim
```